apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -1192,7 +1201,11 @@ def _check_interface_port_operation(self, context, port):
     def _perform_port_operations(self, context):
         # Get port
         port = context.current
-        if port.get('device_owner') == n_constants.DEVICE_OWNER_ROUTER_GW:
+        if not self.fabric_l3:
```

Comment:
Initially I thought that this stuff was all DB-related, but now I see there are some paths that probably should be skipped (e.g. lookup of external segments in APIC config). I'll add some new code to address this.